lib/maestro/aggregate/projection_handler.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Maestro.Aggregate.ProjectionHandler do
+  @moduledoc """
+  `ProjectionHandler`s are used to manage alternate representations of an
+  aggregate.
+
+  This defines a minimal behaviour for use within the aggregate command/event
+  lifecycle. For projections that should be updated immediately iff the relevant
+  events are committed, the relevant `ProjectionHandler` should by included in
+  the list of `:projections` on the aggregate root.
+  """
+
+  @type name :: any()
```

Comment:
It doesn't look like these two typespecs are used any more.

Reply:
Good 👁